portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,49 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="UTF-8">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
-  </head>
-  <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-    </div>
-  </body>
+	<head>
+		<meta charset="UTF-8">
+		<title>Yuyan's Portfolio</title>
+		<link rel="stylesheet" href="style.css">
+		<script src="script.js"></script>
+	</head>
+	<body>
+		<div class="nav">
+			<ul>
+				<li><a class="active" href="index.html">Home</a></li>
+				<li><a href="project.html">Projects</a></li>
+				<li><a href="experience.html">Experiences</a></li>
+			</ul>
+		</div>
+		<div class="profile">
+			<img src="/images/IMG_8092.jpg">
+			<h3>Yuyan Ke</h3>
+			
+		</div>
+		<div id="content">
+			<h2>About Me</h2>
+			<p>
+			Welcome to my portfolio!! I am a rising junior at the School of Engineering and Applied Scientists (SEAS) of Columbia University in the City of New York 
```

Comment:
It's conventional at Google to line break code at 80 characters. In a personal project like this you could choose any convention that you want but I'd prefer (particularly later in the program when the whole team is collaborating) that you line break your code.

Reply:
Do you have any recommendations on how I could have vim tell me the number of characters per line? I tried ":!wc -m %". It gave a total character count per file.

Reply:
I don't use vim. Something like [this hint from StackOverflow](https://stackoverflow.com/questions/235439/vim-80-column-layout-concerns) maybe?